src/connector/postgres.rs

```diff
@@ -547,19 +610,20 @@ impl PostgreSql {
             }
         }));
 
-        // SET NAMES sets the client text encoding. It needs to be explicitly set for automatic
-        // conversion to and from UTF-8 to happen server-side.
-        //
-        // Relevant docs: https://www.postgresql.org/docs/current/multibyte.html
-        let session_variables = format!(
-            r##"
-            {set_search_path}
-            SET NAMES 'UTF8';
```

Comment:
Not needed at all?

Reply:
Are you specifically referring to `SET NAMES UTF8`? It is already set by `tokio-postgres` via connection parameters https://github.com/sfackler/rust-postgres/blob/65c1d146a6b3b91652f211d8c72e776164ee87a4/tokio-postgres/src/connect_raw.rs#L114